README.md

```diff
@@ -0,0 +1,20 @@
+# Puma Cloud Native Buildpack
+
+The Puma CNB sets the start command for a given ruby application that runs on a puma server.
+
+## Integration
+
+It provides `puma` as a dependency, but currently there's no scenario
```

Comment:
I don't think that this statement is true. It doesn't provide any dependency. It just writes a start command.